src/Microsoft.Health.Dicom.Core/Features/Store/DicomDatasetValidator.cs

```diff
@@ -32,9 +34,10 @@ public DicomDatasetValidator(IOptions<FeatureConfiguration> featureConfiguration
             _minimumValidator = minimumValidator;
         }
 
-        public void Validate(DicomDataset dicomDataset, string requiredStudyInstanceUid)
+        public void Validate(DicomDataset dicomDataset, IList<CustomTagEntry> customTagEntries, string requiredStudyInstanceUid)
```

Comment:
I am little worried how this is passed in has a new param.

We need a model that can handle this correctly without passing custom tags like this.

Reply:
how about:  IDictionary<DicomTag,DicomVR> validatingTags..